alternator/executor.cc

```diff
@@ -774,16 +774,16 @@ future<executor::request_return_type> executor::tag_resource(client_state& clien
         co_return api_error::access_denied("Incorrect resource identifier");
     }
     schema_ptr schema = get_table_from_arn(_proxy, rjson::to_string_view(*arn));
```

Comment:
just cosmetics, schema_ptr schema could be now called closer to it's usage (e.g. after 2 ifs below)

btw is `schema` needed here at all, I'd think there is some other way to just get ks and cf

Reply:
The order also determines which order we do error checking - if the request has a bad ARN and also bad tags, which error we show first. I didn't want to change this in the patch.

You're right that the schema is not needed, but this is what get_table_from_arn() currently returns after it gets ks and cf. Since get_table_from_arn() is used in additional places, I though it will be simpler to leave this one function that returns schema.